docs/concepts.md

```diff
@@ -81,47 +80,31 @@ That being said, the operator will still insert child resources even if the pare
 
 Finally, if you wish the operator to manage your pre-existing resources on your service registry, please update all the necessary resources by inserting `owner: cnwan-operator` among their metadata.
 
-## Namespace Lists
+## Monitor namespaces
 
-For the CN-WAN Operator, a namespace can belong to two lists: *allowlist* or *blocklist*. CN-WAN Operator only processes services that belong to namespaces it is allowed to work on: those that are inside the *allowlist*.
+The CN-WAN Operator watches service updates only on *monitored* namespaces. To do so, you need to label a namespace with our reserved label key `operator.cnwan.io/monitor`.
 
-To insert a namespace in a list, you have to label it like this:
+If a namespace is labeled as `operator.cnwan.io/monitor=true` then the operator will watch service updates happening on that namespace. On the contrary, `operator.cnwan.io/monitor=false` will instruct the operator to stay away from that namespace.
 
-```bash
-# Insert namespace in the allowlist
-kubectl label ns namespace-name operator.cnwan.io/allowed=yes
+This being said, you don't need to rush labelling all namespaces as `operator.cnwan.io/monitor=false` in fear of potentially exposing sensitive data on the service registry: namespaces that do not have such label will be ignored by default, as the operator will pretend it is seeing `operator.cnwan.io/monitor=false`. This is useful in case you think you have few namespaces you want to monitor or if you prefer to retain control, even have lots of namespaces to monitor.
 
-# Insert namespace in the blocklist
-kubectl label ns namespace-name operator.cnwan.io/blocked=yes
-```
+Instead, if you have many namespaces to monitor and/or find it tedious to manually do so for every single one of them, you can override this behavior via `monitorNamespacesByDefault: true` on the [operator settings](./configuration.md#monitor-namespaces-by-default): this means that the operator will pretend it is seeing `operator.cnwan.io/monitor=true` and thus watch events in the namespace by default, unless instructed otherwise. This is the opposite scenario from above: now you will need to manually *disable* them.
 
-It doesn't really matter what you put as value (in this case `yes` has been inserted), just as long as the key `operator.cnwan.io/<key>` is as specified above.
+Let's see some examples.
 
-Similarly, to remove a namespace from the list:
+To _monitor_ monitoring on namespace `hr`, do the following:
```

Comment:
I suppose this is a reasult of search and replace "To _enable_ monitoring". I guess you can leave "To _monitor_" or change back "To _enable_ monitoring" to be consistent with the next example below about disabling monitoring.

Reply:
Exactly, thank you for catching this!

Reply:
Solved by amending to

> _enable_ monitoring